scripts/globals/abyssea.lua

```diff
@@ -711,6 +711,11 @@ xi.abyssea.canGiveNMKI = function(mob, dropChance)
 end
 
 xi.abyssea.giveNMDrops = function(mob, player, ID)
+    if not xi.abyssea.mob[mob:getName()] then
+        print("No atma or ki for ", mob:getName())
```

Comment:
looks like this nm legit has neither on retail, which means that can happen without being unimplemented so I think we just want the return.

Reply:
actually that print looks malformed.

Reply:
done